main.js

```diff
@@ -0,0 +1,114 @@
+var weatherCurrent = [];
+var weeklyWeather = [];
+var key = '40a22fcb01995614a7b68804376359eb'
+
+document.querySelector('.search').addEventListener('click', function() {
+  var search = document.querySelector('#search-city').value;
+
+  fetchData(search);
+
+  document.querySelector('#search-city').value = '';
+
+});
+// use entered city and current weather api
+
+var fetchData = function(search) {
+  const urlCurrent = 'https://api.openweathermap.org/data/2.5/weather?q=' + search + '&appid=' + key;
+  fetch(urlCurrent, {
+    method: 'GET',
+    headers: {
+      'Accept': 'application/json'
+    }
+    
+  })
+
```

Comment:
Formatting like this makes it diffcult to read because it's hard to tell what the `then` blocks attach too. Try to setup prettier and you'll see what I mean! Makes your life much easier if you format on save.